supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -20,8 +20,21 @@ Do not rely solely on a shell prompt in a sample command to indicate the require
 If you include a shell prompt to indicate that a user with root privileges must run the command, also include a statement about this requirement in the step text, the introductory text, or the prerequisites.
 ====
 +
-* If a command requires a temporary switch to root privileges, use `sudo` at the beginning of the sample command syntax. If the sample command includes a shell prompt together with `sudo`, the prompt should remain `$` and not be changed to `#`.
-* If multiple commands in a procedure require root privileges, add introductory content to tell the user what to do. For example, "The commands in this procedure require root privileges to run. Either run `su -` to switch to the root user or preface the commands with `sudo`."
+* If a command requires a temporary switch to root privileges, use `sudo` at the beginning of the sample command syntax. 
+If the sample command includes a shell prompt together with `sudo`, the prompt should remain `$` and not be changed to `#`, as in the following example: 
++
+[source,terminal]
+----
+$ sudo systemctl start firewalld
+----
++
+* If multiple commands in a procedure require root privileges, add introductory content to tell the user about the requirement. 
+The following example shows one way that a writer might integrate a requirement for root access into introductory text for a procedure:
++
+.Example AsciiDoc
+----
+A user must perform some of these tasks as the root user or by acquiring effective root privileges using a _setuid_ program, such as `sudo` or `su`.
```

Comment:
> or by acquiring effective root privileges using a _setuid_ program, such as `sudo` or `su`.

The mention of `setuid` here seems a bit out of place. Since we're primarily dealing with the enterprise space, I think we should probably provide guidance to use `sudo` by default, and `su` only at the insistence of the SME reviewing the relevant content. 

For context: 
sudo allows a permitted user to execute a command as the superuser or another user, as specified by the security policy.  Security policies may support credential caching to allow the user to run sudo again for a period of time without requiring authentication.  By default, the sudoers policy caches credentials on a per-terminal basis for 5 minutes. 

su allows commands to be run with a substitute user and group ID. When called with no user specified, su defaults to running an interactive shell as root.

Reply:
I agree that mentioning `setuid` here is not needed, and I've updated the PR to remove it. (I actually took this example from the current RHEL docs, btw.) I don't think it's necessary to go into too much detail about the differences between `su` vs `sudo` here because we link to more in-depth info in the "Additional resources" section. This text is meant as a generic example and is not meant to be prescriptive; it's at the writer's discretion how they want to recommend gaining root privileges.

Reply:
The "Additional resources" info linked says (unfortunately formatted, imo -- nestled under the embedded video but above visual links): 

> But when do you use one, not another? Since the sudo policy is defined in /etc/sudoers, this can give powerful permission controls. **Since sudo can pretty much do everything that su can, I would say it is best to stick with sudo unless you are working with some legacy codes that require the su command.**

>  it's at the writer's discretion how they want to recommend gaining root privileges.

I understand that's how its presented here currently. I don't think leaving it to writer discretion is the most customer-focused approach, particularly without at least a warning that some commands when executed as root (via `su`) can cause catastrophic failure & to take special care to have a SME sign off on its use over `sudo`.